lib/ret_web/controllers/api/v1/scene_controller.ex

```diff
@@ -1,29 +1,71 @@
 defmodule RetWeb.Api.V1.SceneController do
   use RetWeb, :controller
 
-  alias Ret.Scene
-  alias Ret.Repo
+  alias Ret.{Repo, Scene, Storage}
 
-  plug(RetWeb.Plugs.RateLimit when action in [:create])
+  plug(RetWeb.Plugs.RateLimit when action in [:create, :update])
 
-  def show(conn, %{"id" => scene_id}) do
-    scene = Repo.get(Scene, scene_id)
+  def show(conn, %{"id" => scene_sid}) do
```

Comment:
Are we back to using SIDs in APIs?

Reply:
I forgot if we had a discussion about it, but I was just matching the current use of sids for hub objects

Reply:
Okay, yeah, I'm was also waffling on it, so let's just stick with SIDs and UUIDs (i.e. never expose DB IDs in either end-user facing or API endpoints).